--- testtools-2.3.0/testtools/tests/test_run.py.orig	2018-07-26 18:36:38.072223680 +0200
+++ testtools-2.3.0/testtools/tests/test_run.py	2018-07-26 18:56:54.098876462 +0200
@@ -204,10 +204,8 @@
 unittest2.loader._FailedTest.runexample
 Failed to import test module: runexample
 Traceback (most recent call last):
-  File ".../loader.py", line ..., in _find_test_path
-    package = self._get_module_from_name(name)
-  File ".../loader.py", line ..., in _get_module_from_name
-    __import__(name)
+  File ".../loader.py", line ..., in _find_test_path...
+  File ".../loader.py", line ..., in _get_module_from_name...
   File ".../runexample/__init__.py", line 1
     class not in
 ...^...
